lib/new_relic/rack/browser_monitoring.rb

```diff
@@ -37,7 +37,8 @@ def traced_call(env)
       if (js_to_inject != "") && should_instrument?(env, status, headers)
         response_string = autoinstrument_source(response, headers, js_to_inject)
         if headers.key?(CONTENT_LENGTH)
-          headers[CONTENT_LENGTH] = response_string.size.to_s
+          content_length = response_string ? response_string.size : 0
```

Comment:
```suggestion
          content_length = response_string ? response_string.bytesize : 0
```

`size` only return the character length, we need to length in bytes for this to be correct

Yes, this error exist in the original PR about this issue: https://github.com/newrelic/rpm/commit/5168ee07acb4d00fbbe507720c1bfd9fb8244c23 (https://github.com/newrelic/rpm/pull/312)

I'm trying out newrelic_rpm 6.9.0.363 in my app and `</html>` is missing its end tag:

<img width="434" alt="Screenshot 2020-03-09 at 18 21 20" src="https://user-images.githubusercontent.com/42626/76240012-cd459e00-6232-11ea-8021-a286c516190e.png">

Reply:
Created https://github.com/newrelic/rpm/issues/325 about it so it isn't forgotten

Reply:
Ah, there's already another PR making this change: https://github.com/newrelic/rpm/pull/322